lib/pf/Switch.pm

```diff
@@ -2732,6 +2732,9 @@ sub returnRadiusAccessAccept {
 
     my $radius_reply_ref = {};
 
+    # should this node be kicked out?
+    $self->returnRadiusDeny($args);
```

Comment:
should be named handleRadiusDeny since this does not always return it but evaluates if it should and then does it